cmd/avalanche.go

```diff
@@ -13,6 +14,26 @@ import (
 	kingpin "gopkg.in/alecthomas/kingpin.v2"
 )
 
+type ignoredIntervals map[string]bool
+
+func (i ignoredIntervals) Set(value string) error {
+	fields := strings.Split(value, ",")
+	for _, f := range fields {
+		i[f] = true
+	}
+	return nil
+}
+
+func (i ignoredIntervals) String() string {
+	return ""
+}
+
+func IgnoredIntervals(s kingpin.Settings) (target map[string]bool) {
+	target = make(map[string]bool)
+	s.SetValue((ignoredIntervals)(target))
+	return
```

Comment:
the function signature shows a return type of `map[string]bool` but it does not seem like we are returning anything?